spec.html

```diff
@@ -187,6 +306,33 @@ <h1>Runtime Semantics: HostInitializeSyntheticRealm ( _realm_ )</h1>
 				global object.
 			</p>
 		</emu-clause>
+
+		
+		<emu-clause id="sec-hostimportmodulebindingdynamically" aoid="HostImportModuleBindingDynamically">
+			<h1>HostImportModuleBindingDynamically ( _referencingScriptOrModule_, _specifier_, _bindingName_, _promiseCapability_ )</h1>
+			<p>The host-defined abstract operation HostImportModuleBindingDynamically takes arguments _referencingScriptOrModule_ (a Script Record or Module Record or *null*), _specifier_ (a |ModuleSpecifier| String), _bindingName_ (a |IdentifierName| String) and _promiseCapability_ (a PromiseCapability Record). It performs any necessary setup work in order to make available the module corresponding to _specifier_ occurring within the context of the script or module represented by _referencingScriptOrModule_. _referencingScriptOrModule_ is *null*. It then performs FinishDynamicImport to finish the dynamic import process.</p>
+			<p>The implementation of HostImportModuleBindingDynamically must conform with the implementation of HostImportModuleDynamically first and foremost, second, it should conform with the following:</p>
+			<li>
+				The host environment must conform to one of the two following sets of requirements:
+				<dl>
+				<dt>Success path</dt>
+
+				<dd>
+					<ul>
+						<li>The completion value must be the exported value for the _bindingName_ associated to the Module Namespace Object.</li>
```

Comment:
This needs to add a step to include a resolution of `GetWrappedValue` over the completion value. It might have a promise wrapping the resolution of `FinishDynamicImport` to capture the values.

It might need to dupe the received promiseCapability to unwrap the binding values from the resolved namespace.

It also needs to guarantee the error is from this incubator realm.

Reply:
you're correct, that's missing.

Reply:
fixed!